### PR TITLE
Fix injector timeouts

### LINKED ISSUE #748: Worker hangs up and drakrun waits forever for injection to finish
**Describe the bug**

Drakrun worker can't recover from Inject.CreateProc `cmd.exe /c ipconfig /release >nul` when it takes forever due to tenporary DHCP connection issues

I see that there is `timeout=120` set in provisioning part: https://github.com/CERT-Polska/drakvuf-sandbox/blob/5017f9ca485d409012db165139b1905c163c445d/drakrun/drakrun/main.py#L608-L615

but `timeout` argument handling is not implemented: https://github.com/CERT-Polska/drakvuf-sandbox/blob/5017f9ca485d409012db165139b1905c163c445d/drakrun/drakrun/injector.py#L68-L73

**Output of the [status checking commands](https://github.com/CERT-Polska/drakvuf-sandbox#checking-service-status)**

```
Dec 07 12:29:01 ... drakrun[8661]: DRAKVUF injector v0.8-git+-1 Copyright (C) 2014-2021 Tamas K Lengyel
Dec 07 12:29:01 ... drakrun[8661]: tcpdump: listening on vif30446.0-emu, link-type EN10MB (Ethernet), capture size 262144 bytes
Dec 07 12:29:01 ... drakrun[8661]: [2022-12-07 12:29:01,218][INFO] Using command: regsvr32 /s C:\Users\...\Desktop\malwar.dll
Dec 07 12:29:01 ... drakrun[8661]: INFO:karton.drakrun-oss:Using command: regsvr32 /s C:\Users\...\Desktop\malwar.dll
Dec 07 12:29:01 ... drakrun[8661]: [2022-12-07 12:29:01,221][INFO] Setting up network...
Dec 07 12:29:01 ... drakrun[8661]: INFO:karton.drakrun-oss:Setting up network...
Dec 07 12:29:01 ... drakrun[8661]: DRAKVUF injector v0.8-git+-1 Copyright (C) 2014-2021 Tamas K Lengyel
Dec 07 12:29:01 ... drakrun[8661]: {"Plugin": "inject", "TimeStamp": "1670412541.519323", "Method": "CreateProc", "Status": "Success", "ProcessName": "", "Arguments": "/C ipconfig /release >nul", "InjectedPid": 2800, "InjectedTid": 2804}
Dec 07 12:29:01 ... drakrun[8661]: DRAKVUF injector v0.8-git+-1 Copyright (C) 2014-2021 Tamas K Lengyel
```

### PR DESCRIPTION
**Pull request by BonusPlay**
Should close #748.

### CODE REVIEW

File: drakrun/drakrun/injector.py [hunk 1]
```diff
@@ -52,22 +62,40 @@ def _get_cmdline_createproc(self, exec_cmd: str, wait: bool = False) -> List[str
     def write_file(
         self, local_path: str, remote_path: str, timeout: int = 60
     ) -> subprocess.CompletedProcess:
-        """Copy local file to the VM"""
-        injector_cmd = self._get_cmdline_writefile(local_path, remote_path)
+        """
+        Copy local file to the VM
+        we pass (timeout-5) to drakvuf to give it 5 seconds to finish it's loop
```
Comment (msm-code) [inline drakrun/drakrun/injector.py:67]:
```suggestion
        we pass (timeout-5) to drakvuf to give it 5 seconds to finish its loop
```

File: drakrun/drakrun/injector.py [hunk 2]
```diff
@@ -52,22 +62,40 @@ def _get_cmdline_createproc(self, exec_cmd: str, wait: bool = False) -> List[str
...
         )
 
     def read_file(
         self, remote_path: str, local_path: str, timeout: int = 60
     ) -> subprocess.CompletedProcess:
-        """Copy VM file to local"""
-        injector_cmd = self._get_cmdline_readfile(remote_path, local_path)
+        """
+        Copy VM file to local
+        we pass (timeout-5) to drakvuf to give it 5 seconds to finish it's loop
```
Comment (msm-code) [inline drakrun/drakrun/injector.py:82]:
```suggestion
        we pass (timeout-5) to drakvuf to give it 5 seconds to finish its loop
```

File: drakrun/drakrun/injector.py [hunk 3]
```diff
@@ -52,22 +62,40 @@ def _get_cmdline_createproc(self, exec_cmd: str, wait: bool = False) -> List[str
     def write_file(
         self, local_path: str, remote_path: str, timeout: int = 60
     ) -> subprocess.CompletedProcess:
-        """Copy local file to the VM"""
-        injector_cmd = self._get_cmdline_writefile(local_path, remote_path)
+        """
+        Copy local file to the VM
+        we pass (timeout-5) to drakvuf to give it 5 seconds to finish it's loop
+        """
+        drakvuf_timeout = timeout - 5 if timeout != 0 else 0
```
Comment (msm-code) [inline drakrun/drakrun/injector.py:69]:
```suggestion
        drakvuf_timeout = timeout - 5 if timeout > 5 else 0
```
shouldn't it look like this?

File: drakrun/drakrun/injector.py [hunk 4]
```diff
@@ -52,22 +62,40 @@ def _get_cmdline_createproc(self, exec_cmd: str, wait: bool = False) -> List[str
...
     def read_file(
         self, remote_path: str, local_path: str, timeout: int = 60
     ) -> subprocess.CompletedProcess:
-        """Copy VM file to local"""
-        injector_cmd = self._get_cmdline_readfile(remote_path, local_path)
+        """
+        Copy VM file to local
+        we pass (timeout-5) to drakvuf to give it 5 seconds to finish it's loop
+        """
+        drakvuf_timeout = timeout - 5 if timeout != 0 else 0
```
Comment (msm-code) [inline drakrun/drakrun/injector.py:84]:
```suggestion
        drakvuf_timeout = timeout - 5 if timeout > 5 else 0
```

**Comment by BonusPlay** [comment]
CI passes, needs to be re-checked when #784 is merged

**Review by chivay — APPROVED** [review]

**Pull request merged**